Berksfile

```diff
@@ -0,0 +1,6 @@
+source 'https://supermarket.getchef.com'
+
+metadata
+
+cookbook 'apt'
```

Comment:
These should get wrapped into an integration group since they're only for testing